src/Tasklet.cpp

```diff
@@ -212,6 +215,7 @@ bool Tasklet::Insert()
 
 bool Tasklet::SwitchImplementation()
 {
+	TelemetryZone telemetryZone(TMCM_CPP, "Tasklet::SwitchImplementation()", __FILE__, __LINE__, tracy::Color::LightGreen);
```

Comment:
Just checking what information, you are hoping from this tag.

This function is related specifically to the `tasklet.switch` command.

Reply:
Basically same motivation as for the zone in `ScheduleManager::RunNTasklets`. This may or may not be useful.